SpinCore_pp/beta_conv.py

```diff
@@ -0,0 +1,78 @@
+from pylab import array
+import numpy as np
+from pyspecdata import r_, nddata
+
+
+def prog_beta(desired_actual):
+    """
+    Takes the desired beta (us *sqrt(W)) and tells the
+    user what beta should be programmed in order to get the actual desired
+    beta
+    Parameters
+    ==========
+    desired_actual: float
+                    the actual beta you wish the spincore to output,
+                    in us sqrt(W)
+    Returns
+    =======
+    retval: float
+            The beta you tell spincore in order to get the desired actual.
+    """
+    # {{{ list of programmed beta, and actual beta - used in
+    # generating the calibrated fit
+    datapoints = [
+        (1, 2.25869),
+        (2, 5.78065),
+        (3, 10.4132),
+        (3.5, 13.2053),
+        (4, 16.9808),
+        (4.5, 20.4275),
+        (5, 24.8538),
+        (5.5, 30.0159),
+        (6, 35.6903),
+        (6.5, 42.2129),
+        (7, 48.9577),
+        (7.5, 55.977),
+        (8, 63.0445),
+        (8.5, 69.881),
+        (9, 76.9727),
+        (9.5, 84.51),
+        (10, 91.700),
+        (10.5, 98.63),
+        (11, 105.327),
+        (11.5, 112.9),
+        (12, 120.264),
+        (12.5, 126.2),
```

Comment:
these will be updated based on the output of the example script to capture the pulses